lineapy/plugins/ray_pipeline_writer.py

```diff
@@ -114,6 +114,11 @@ def _write_operators(
     def docker_template_name(self) -> str:
         return "ray_dockerfile.jinja"
 
+    def _get_requirements(self):
+        libraries = super()._get_requirements()
+        libraries["packaging"] = "21.3"
```

Comment:
This is because `packaging` is required to run the DAG, but not any of the tasks. The inherited `_get_requirements` basically aggregates all the libraries needed to run the tasks. `packaging` is not needed for any task but is needed for [an extra check in the ray DAG for library version](https://github.com/LineaLabs/lineapy/blob/5571fcd6f57482191ce3b19fc66bf223069c6d1f/lineapy/plugins/jinja_templates/ray_dag_workflow.jinja#L6) so it needs to be added in afterwards

Reply:
Added comment